### ISSUE TITLE
Missing Nullable type modifiers in definition of JsonLdOptions

### ISSUE THREAD

**Issue by kasei** [w3c/json-ld-api#359]
[The JsonLdOptions Type](https://w3c.github.io/json-ld-api/#the-jsonldoptions-type) section includes a WebIDL definition of `JsonLdOptions` that includes two `USVString` values and one `LoadDocumentCallback` value which default to `null`. I don't believe these types can be asserted as `null`, and think they need to be updated to `Nullable` equivalents (`USVString?` and `LoadDocumentCallback?`):

```
  LoadDocumentCallback?   documentLoader = null;
  USVString?              processingMode = null;
  USVString?              rdfDirection = null;
```

**Comment by gkellogg** [comment]
Fixed in #362. Also, default `processingMode` to "json-ld-1.1".

**Comment by kasei** [comment]
Looks good, thanks!
